src/main/resources/templates/welcome.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Welcome to GilfGang website</title>
+        <link href='https://unpkg.com/css.gg@2.0.0/icons/css/globe-alt.css' rel='stylesheet'>
+        <link href='https://unpkg.com/css.gg@2.0.0/icons/css/arrow-right-r.css' rel='stylesheet'>
+        <link href='https://unpkg.com/css.gg@2.0.0/icons/css/arrow-left-r.css' rel='stylesheet'>
+    <link rel="stylesheet" type="text/css" href="/static/styles.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="/static/styles.css">
```
Why do you need type="text/css" ?